src/App.tsx

```diff
@@ -1,93 +1,79 @@
-            </div>
-            <input type="text" className="edit" />
-          </li>
+    default:
+      return todos;
+  }
+}
 
-          <li className="completed">
-            <div className="view">
-              <input type="checkbox" className="toggle" id="toggle-completed" />
-              <label htmlFor="toggle-completed">qwertyuio</label>
-              <button type="button" className="destroy" data-cy="deleteTodo" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
-
-          <li className="editing">
-            <div className="view">
-              <input type="checkbox" className="toggle" id="toggle-editing" />
-              <label htmlFor="toggle-editing">zxcvbnm</label>
-              <button type="button" className="destroy" data-cy="deleteTodo" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
+export const App: React.FC = () => {
+  const todos = useContext(TodosContext);
+  const onChange = useContext(DispatchContext);
+  const [filteredField, setFilteredField] = useState('all');
+  const filteredTodos = getFilteredTodos(todos, filteredField);
+  const unCompletedTodo = todos.filter((todo => !todo.completed));
+  const completedTodo = todos.filter((todo => todo.completed));
 
-          <li>
-            <div className="view">
-              <input type="checkbox" className="toggle" id="toggle-view2" />
-              <label htmlFor="toggle-view2">1234567890</label>
-              <button type="button" className="destroy" data-cy="deleteTodo" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
-        </ul>
-      </section>
+  return (
+    <>
+      <div className="todoapp">
+        <Header />
 
-      <footer className="footer">
-        <span className="todo-count" data-cy="todosCounter">
-          3 items left
-        </span>
+        {todos.length !== 0
+          && (
+            <>
+              <section className="main">
+                {todos.length !== 0 && (
+                  <>
+                    <input
+                      type="checkbox"
+                      id="toggle-all"
+                      className="toggle-all"
+                      data-cy="toggleAll"
+                      onChange={() => onChange({ type: 'toggle-all' })}
+                    />
+                    <label htmlFor="toggle-all"> </label>
+                  </>
+                )}
+                <TodoList items={filteredTodos} />
+              </section>
 
-        <ul className="filters">
-          <li>
-            <a href="#/" className="selected">All</a>
-          </li>
+              <footer className="footer">
+                <span className="todo-count" data-cy="todosCounter">
+                  {`${unCompletedTodo.length} items left`}
+                </span>
+                <TodosFilter sortBy={setFilteredField} />
+                {completedTodo.length !== 0 && (
```

Comment:
```suggestion
                {!!completedTodo.length && (
```
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**